packages/commons/src/config/consumerServiceConfig.ts

```diff
@@ -3,7 +3,17 @@ import { AWSConfig } from "./awsConfig.js";
 import { KafkaConfig } from "./kafkaConfig.js";
 import { ReadModelDbConfig } from "./readmodelDbConfig.js";
 
-export const KafkaConsumerConfig = KafkaConfig.and(AWSConfig);
+export const KafkaConsumerConfig = KafkaConfig.and(AWSConfig).and(
+  z
+    .object({
+      MESSAGE_STARTING_OFFSET: z
```

Comment:
a bit more clear when setting configurations
```suggestion
      TOPIC_STARTING_OFFSET: z
```